internify/src/store/reducers/jobPostingReducer.js

```diff
@@ -1,12 +1,31 @@
 
-const initialState = {
-  isDataLoading: false,
-  finalJobPostingsList: [],
+// const initialState = {
```

Comment:
do we need these comments still?

Reply:
sorry - must have missed deleting that last time round!